config/service_providers.yml

```diff
@@ -22,6 +22,13 @@ development:
       cert: 'sample_cert'
       fingerprint: 'F9:A3:9B:2F:8F:1C:E2:79:27:69:EB:32:ED:2A:D5:A2:A7:58:5F:C0:74:8A:4A:03:D9:0F:77:A5:89:7F:F9:68'
 
+    'https://dashboard.login.gov':
+      metadata_url: 'http://localhost:3000/users/auth/saml/metadata'
+      acs_url: 'http://localhost:3000/users/auth/saml/callback'
+      assertion_consumer_logout_service_url: 'http://localhost:3000/users/auth/saml/logout'
+      sp_initiated_login_url: 'http://localhost:3000/users/auth/saml'
```

Comment:
I see. I missed the part where this is only to be used in the `development` environment. My only concern is if there's the potential of port confusion. Perhaps the dashboard should run on port 8080 and its README should be updated to mention that port that way it's clear which app is running on which port. Thoughts?